Package.swift

```diff
@@ -5,11 +5,18 @@ import PackageDescription
 
 let package = Package(
     name: "SQLiteVec",
+    platforms: [
+        .iOS(.v16)
+    ],
```

Comment:
I wouldn't add it. Specifying `platforms` (according to the [docs](https://github.com/swiftlang/swift-package-manager/blob/main/Documentation/PackageDescription.md#supportedplatform)) means

> By default, the Swift Package Manager assigns a predefined minimum deployment version for each supported platforms unless you configure supported platforms using the platforms API. 

So it should work on iOS without explicitly specifying it here

Reply:
Ya I was curious about that. Yup not problem, I'll remove